### PR TITLE
Add Math.hypot binding

### PR DESCRIPTION
**Pull request by twilco**
Adds `Math.hypot()` binding for #275.

### CODE REVIEW

**Comment by alexcrichton** [comment]
Awesome, thanks!

File: crates/js-sys/src/lib.rs [hunk 1]
```diff
@@ -1154,6 +1154,12 @@ extern "C" {
     #[wasm_bindgen(static_method_of = Math)]
     pub fn fround(x: f64) -> f32;
 
+    /// The Math.hypot() function returns the square root of the sum of squares of its arguments.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/fround
```
Comment (jontro) [inline crates/js-sys/src/lib.rs]: Shouldn't this be https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/hypot ?

Comment (alexcrichton) [reply crates/js-sys/src/lib.rs]: Ah yes indeed! @jontro want to file a follow-up PR?

Comment (jontro) [reply crates/js-sys/src/lib.rs]: Done! https://github.com/rustwasm/wasm-bindgen/pull/567

**Pull request merged**